helpers/postwomanTesting.js

```diff
@@ -144,4 +144,40 @@ class Expectation {
       ? this._pass()
       : this._fail(this._fmtNot(`Expected ${this.expectValue} to (not)be 500-level status`))
   }
+  toHaveLength(expectedLength) {
+    const actualLength = this.expectValue.length
+    return this._satisfies(actualLength, expectedLength)
+      ? this._pass()
+      : this._fail(
+          this._fmtNot(
+            `Expected length to be ${expectedLength} but actual length was ${actualLength}`
+          )
+        )
+  }
+  toBeOfType(expectedType) {
```

Comment:
```suggestion
  toBeType(expectedType) {
```

Reply:
`toBeType` sounds much better than `toBeOfType`

Reply:
Well I'm not a native English speaker but "to be of type" would be grammatically correct I believe. But as long as it's clear in the docs I guess it doesn't matter.